src/createSliderWithTooltip.jsx

```diff
@@ -26,13 +28,25 @@ export default function createSliderWithTooltip(Component) {
       });
     }
     handleWithTooltip = ({ value, dragging, index, disabled, ...restProps }) => {
-      const { tipFormatter } = this.props;
+      const {
+        tipFormatter,
+        tipProps,
+      } = this.props;
+
+      const {
+        prefixCls = 'rc-slider-tooltip',
+        overlay = tipFormatter(value),
+        placement = 'top',
```

Comment:
Sorry, I have two question.
1. what is difference? `placement` is already have default value. is it convention?
2. how about `prefixCls`? I think `prefixCls` should have same form with `placement` in your suggestion. like 
```
placement='top'
prefixCls='rc-slider-tooltip'
{...restTooltipProps}
overlay={tipFormatter(value)}
overlay={overlay} 
visible={!disabled && (this.state.visibles[index] || dragging)}
key={index}
```